content/misc/lab-setup.md

```diff
@@ -0,0 +1,34 @@
+# Setting up the Lab Environment
+
+If you have already cloned the repository, make sure it is updated:
+
+```console
+cd operating-systems
+git pull --rebase
```

Comment:
```suggestion
student@os:~$ cd operating-systems

student@os:~/operating-systems$ git pull --rebase
```
It's better to leave an empty line between subsequent commands to make them more clearly distinguishable.